kratos/python/add_deprecated_variables_to_python.cpp

```diff
@@ -140,6 +140,16 @@ void  AddDeprecatedVariablesToPython()
 
     KRATOS_REGISTER_IN_PYTHON_3D_VARIABLE_WITH_COMPONENTS( xi_c )
 
+	//for Thermo Mechanical Application
+	KRATOS_REGISTER_IN_PYTHON_VARIABLE(NODE_PROPERTY_ID)
+	KRATOS_REGISTER_IN_PYTHON_VARIABLE(REF_ID)
+	KRATOS_REGISTER_IN_PYTHON_VARIABLE(PARTICLE_RADIUS)
+	KRATOS_REGISTER_IN_PYTHON_VARIABLE(POSETIVE_DISTANCE)
```

Comment:
posEtive? Just remarking, I don't really mind.

Reply:
Yep, its thit way in the application as well, we can make a different issue for that one.